lib/user_agent_parser/processor.ex

```diff
@@ -0,0 +1,29 @@
+defmodule UserAgentParser.Processor do
+
+  alias UserAgentParser.Storage
+
+  def sanitize(user_agent) do
+    safe =
+      user_agent
+      |> String.trim
+
```

Comment:
Is there a preferred style for the newline here? I feel like its made clear that the return is happening due to the lack of indentation. Don't have a strong opinion, just curious.